src/main/java/uk/gov/hmcts/ccd/domain/model/casedeletion/TTL.java

```diff
@@ -27,6 +28,40 @@ public class TTL {
     private String suspended;
 
     public boolean isSuspended() {
-        return suspended != null && !suspended.equals(NO);
+        return suspended != null && !suspended.equalsIgnoreCase(NO);
     }
```

Comment:
Can we use lombok for equals and hasCode?  unless we have a custom definition of equals and hashcode

Reply:
We are using a custom one for equals due to behaviour of suspended flag.